### PR TITLE
Fixed code indent and padding

### PR DESCRIPTION
**Pull request by tuantinghuang**
Relate #349 #345 

Screenshot 2023-08-02 at 3 26 39 PM

### CODE REVIEW

**Comment by shiffman** [comment]
This looks great! One small issue now, however, is that the vertical padding is such that when there is code text on two lines one above the other, they blend together. Maybe we should reduce this by a pixel or two?

image

Also, what do you think about extending the grey code block highlighting by a little bit?

image

File: magicbook/stylesheets/components/typography.scss
Comment (tuantinghuang) [inline magicbook/stylesheets/components/typography.scss:1]: Adding some padding to the grey code blocks and fixed the inline code's vertical padding.

Screenshot 2023-08-09 at 8 29 06 PM

Comment (shiffman) [reply magicbook/stylesheets/components/typography.scss:1]: This is looking great!

**Comment by shiffman** [comment]
Merging! We can close #345 and #349 but maybe we want to wait and check how it looks after finalize #348.

**Pull request merged**